packages/plans-grid/src/plans-table/index.tsx

```diff
@@ -23,6 +23,10 @@ export interface Props {
 	currentDomain?: DomainSuggestions.DomainSuggestion;
 	disabledPlans?: { [ planSlug: string ]: string };
 	locale: string;
+	showTaglines?: boolean;
+	CTAVariation?: 'FULL_WIDTH' | 'NORMAL';
+	popularBadgeVariation: 'ON_TOP' | 'NEXT_TO_NAME';
+	customTagLines?: Record< string, string >;
```

Comment:
By looking at [how this prop is used](https://github.com/Automattic/wp-calypso/pull/47373/files#diff-bd581c1ba298f6955b07ee66bc44113010bb9638b67b8a6e1770a6ea1ff248aeR59), we probably want keys to be typed as plan slugs.